cleantalk.antispam/options.php

```diff
@@ -77,6 +61,7 @@
         }
 
         // Send empty feedback for version comparison in Dashboard
+        $new_key = $new_key ?? $old_key;
```

Comment:
Replace to "isset()" instead "??" - to support php 5.6

Reply:
Fixed https://github.com/CleanTalk/bitrix-antispam/pull/21/commits/f273a9a26e71d385d31559df5efe572b2675a129